cancelRelease.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
Would you be fine to call this script abort or something else then cancel? I thought we cancel the process instance (via cancel command) and was wondering why we need the date :laughing: Then I saw we send a message.

Reply:
That makes sense! Great suggestion 👍